cmd/start.go

```diff
@@ -159,11 +162,18 @@ func ingest(ctx context.Context, bheUrl url.URL, bheClient *http.Client, in <-ch
 		}
 
 		if req, err := rest.NewRequest(ctx, "POST", endpoint, body, nil, nil); err != nil {
-			log.Error(err, "unable to create request")
-		} else if _, err := bheClient.Do(req); err != nil {
-			log.Error(err, "unable to send data to bloodhound enterprise", "bheUrl", bheUrl)
+			return fmt.Errorf("unable to create ingest HTTP request")
+		} else if response, err := bheClient.Do(req); err != nil {
+			return fmt.Errorf("unable to send data to BHE ingest endpoint %v: %v", bheUrl, err)
+		} else if response.StatusCode != http.StatusAccepted {
+			if bodyBytes, err := io.ReadAll(response.Body); err != nil {
+				return fmt.Errorf("BHE returned HTTP status %d(%s). Failure reading response body: %v", response.StatusCode, response.Status, err)
```

Comment:
For all uses of `fmt.Errorf` that use an `error` as a formatting parameter we should always prefer `%w` as this will create a wrapper error for us that maintains a reference to the original error. I'm only commenting on this particular line but we should replace all usages of `%v` for formatting `error` types in the surrounding code.